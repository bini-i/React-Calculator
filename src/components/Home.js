import React from "react";

const Home = () => (
  <>
    <h2>Welcom to our page</h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque iusto
      nisi officiis, quaerat adipisci labore rem odit distinctio dignissimos
      impedit quae magni nemo id? Repudiandae odio ad libero ab dolorem.
    </p>
  </>
);

export default Home;
