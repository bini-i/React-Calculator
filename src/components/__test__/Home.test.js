/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import Home from "../Home";

describe("snapshot test for Home component", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
