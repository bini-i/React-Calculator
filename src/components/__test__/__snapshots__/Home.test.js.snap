// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for App component renders correctly 1`] = `
Array [
  <span>
    0
  </span>,
  <div>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      %
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#F5913E",
          "width": "25%",
        }
      }
      type="button"
    >
      รท
    </button>
  </div>,
  <div>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      9
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#F5913E",
          "width": "25%",
        }
      }
      type="button"
    >
      x
    </button>
  </div>,
  <div>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      6
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#F5913E",
          "width": "25%",
        }
      }
      type="button"
    >
      -
    </button>
  </div>,
  <div>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      3
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#F5913E",
          "width": "25%",
        }
      }
      type="button"
    >
      +
    </button>
  </div>,
  <div>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "50%",
        }
      }
      type="button"
    >
      0
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#E0E0E0",
          "width": "25%",
        }
      }
      type="button"
    >
      .
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#F5913E",
          "width": "25%",
        }
      }
      type="button"
    >
      =
    </button>
  </div>,
]
`;

exports[`snapshot test for Home component renders correctly 1`] = `
Array [
  <h2>
    Welcom to our page
  </h2>,
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque iusto nisi officiis, quaerat adipisci labore rem odit distinctio dignissimos impedit quae magni nemo id? Repudiandae odio ad libero ab dolorem.
  </p>,
]
`;
